modules/client-configuration/pages/snippets/trust_gpg.adoc

```diff
@@ -9,8 +9,7 @@ However, you cannot install new client tool packages or update them until the ke
 Salt clients use now GPG key information entered for a software channel to manage the trusted keys.
 When a software channel with GPG key information is assigned to a client, the key gets trusted as soon as the channel is refreshed or the first package gets installed from this channel.
 
-// The meaning of the following sentence is not clear
-The GPG key URL which is set of a software channel must exist.
+The GPG key URL parameter in the software channel page can now contain multiple key urls separated by "whitespace".
```

Comment:
```suggestion
The GPG key URL parameter in the software channel page can contain multiple key URLs separated by "whitespace".
```
I would specify a time such as "now" (release notes could carry this info). If really needed we could specify an Uyuni version number.